plugins/package-managers/python/src/main/kotlin/utils/PythonInspector.kt

```diff
@@ -111,7 +111,9 @@ internal object PythonInspector : CommandLineTool {
             val packagePurls = mutableSetOf<String>()
             binaryResult.projects.forEach { project ->
                 project.packageData.forEach { data ->
-                    data.dependencies.mapTo(packagePurls) { it.purl }
+                    data.dependencies
+                        .mapNotNull { it.purl }
+                        .toCollection(packagePurls)
```

Comment:
I missed that one on the first review, it should also be `mapNotNullTo(packagePurls) { it.purl }